src/index.ts

```diff
@@ -3,6 +3,7 @@ export {MonacoEditorModule} from './monaco-editor.module';
 
 // Directives
 export {MonacoEditorDirective} from './directives/monaco-editor/monaco-editor.directive';
+export {MonacoEditorService} from './services/monaco-editor.service';
```

Comment:
Put this under a `// Services` "section"

Reply:
Will do